From bf4819c9134716da67a7e294ba2816a30096424a Mon Sep 17 00:00:00 2001
From: Alfred Chen <cchalpha@gmail.com>
Date: Tue, 19 Feb 2019 10:23:04 +0800
Subject: [PATCH 69/70] pds: [Sync] 977e4be5eb71 x86/stackprotector: Remove the
 call to boot_init_stack_canary() from cpu_startup_entry()

---
 kernel/sched/pds_sched.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/kernel/sched/pds_sched.h b/kernel/sched/pds_sched.h
index 45db698bd811..02e1381b23e5 100644
--- a/kernel/sched/pds_sched.h
+++ b/kernel/sched/pds_sched.h
@@ -30,7 +30,6 @@
 #include <linux/proc_fs.h>
 #include <linux/psi.h>
 #include <linux/slab.h>
-#include <linux/stackprotector.h>
 #include <linux/stop_machine.h>
 #include <linux/suspend.h>
 #include <linux/swait.h>
-- 
2.21.0

